pkg/plugin/trivy/plugin.go

```diff
@@ -172,8 +174,9 @@ func (c Config) setResourceLimit(configKey string, k8sResourceList *corev1.Resou
 }
 
 type plugin struct {
-	clock       ext.Clock
-	idGenerator ext.IDGenerator
+	clock          ext.Clock
+	idGenerator    ext.IDGenerator
+	objectResolver *kube.ObjectResolver
```

Comment:
I'd add it in the commit where we actually use it. Here, according to the change description, we are only changing the interface of vulnerability plugins.

Reply:
I thought the same thing during the change, but there was some dependent change needed in test cases as well so included it in this PR. Anyway reverting it, we will add it in another PR. thanks.